project3/styles1.css

```diff
@@ -22,12 +22,14 @@ h2 {
 }
```

Comment:
I can't comment on the exact line -- sorry! The `padding` rule on line 6 is overriding the `padding-bottom: 100px;` rule you have on line 3. Either delete line 6 or delete _both_ padding rules (since there is no default padding on a `header` element, without line 3, line 6 is unnecessary).